pallets/collator-assignment/src/lib.rs

```diff
@@ -233,6 +233,14 @@ pub mod pallet {
                     max_collators: collators_per_parathread,
                 });
             }
+
+            // Prioritize paras by tip
+            // This doesn't distinguish between parachains and parathreads
+            chains.sort_by(|a, b| {
+                T::CollatorAssignmentTip::get_para_tip(b.para_id)
+                    .cmp(&T::CollatorAssignmentTip::get_para_tip(a.para_id))
+            });
```

Comment:
Can the fee be `None` here? Because I believe the order will be `Some(1) > Some(0) > None`. Which I guess is ok? Add a test for this case if you haven't already.

Reply:
Yes, by default it's `None`, and the order will be as you described. If all are `None`, then the order shouldn't change